assets/bingo/lang/en_us.json

```diff
@@ -27,10 +27,15 @@
 	"bingo.logging.error": "ERROR",
 	"bingo.setup.error.unknown_category": "Item %1$s has category %2$s, but category %2$s could not be found.",
 
-	"bingo.game.go_to_lobby": "%s to go back to the lobby.\n%s to switch to spectator mode.",
-	"bingo.game.go_to_lobby.hint": "Hint: You can also use the command %s at any time manually to return to the lobby or %s for spectator mode.",
-	"bingo.game.go_to_lobby.click_lobby": "Click here",
-	"bingo.game.go_to_lobby.click_spectator": "Click here",
+	"bingo.game.menu.text": "[%s]",
+	"bingo.game.menu.icon": "≡",
```

Comment:
I was thinking that perhaps in some languages they may want to change the icon to something else. 

Otherwise, I can just hardcode the icon in the datapack?

Reply:
I think the icon should be universal, I'd hardcode it for simplicity